org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -206,6 +218,38 @@ public IProxyService getProxyService() {
 		return null;
 	}
 
```

Comment:
JavaLanguageServerPlugin is probably not the right place for this. I'd suggest to have this in the `org.eclipse.jdt.ls.debug` plugin.
@testforstephen also has  code to find a type. Maybe create a 'SearchUtil' .